_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
+| {% include anchor a="error.consumption.attributes.predecessorIsNotPeerSharedRelationshipAttribute" %}           | The predecessor is not a [peer shared RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                             |
+| {% include anchor a="error.consumption.attributes.predecessorIsNotRepositoryAttribute" %}                       | The predecessor is not a [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                                           |
+| {% include anchor a="error.consumption.attributes.predecessorIsNotThirdPartyOwnedRelationshipAttribute" %}      | The predecessor is not a [third party owned RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                       |
+| {% include anchor a="error.consumption.attributes.predecessorSourceAttributeIsNotRepositoryAttribute" %}        | The predecessor source Attribute is not a [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                          |
+| {% include anchor a="error.consumption.attributes.predecessorSourceContentIsNotEqualToCopyContent" %}           | The predecessor source Attribute `content` doesn't match the content of the predecessor shared Attribute copy. ([Update Attributes by succession]({% link _docs_integrate/update-attributes-by-succession.md %}))                                                                                           |
+| {% include anchor a="error.consumption.attributes.senderIsNotPeerOfSharedAttribute" %}                          | The sender of the Attribute is not the peer you shared the Attribute with.                                                                                                                                                                                                                                  |
+| {% include anchor a="error.consumption.attributes.setPredecessorIdDoesNotMatchActualPredecessorId" %}           | The predecessor's `id` and the explicitly set value for the successor's succeeds field don't match.                                                                                                                                                                                                         |
```

Comment:
The formatting "`succeeds`" can be used here for this property. 

Same for other places (missing formatting of properties).